core/src/test/java/nl/nn/adapterframework/doc/DocWriterNewIntegrationTest.java

```diff
@@ -126,4 +126,14 @@ public void testStatistics() throws IOException {
 			writer.close();
 		}
 	}
+
+	@Test
+	public void testLoggingWithExtraExceptionArgument() {
+		try {
+			throw new IllegalStateException();
+		} catch(IllegalStateException e) {
+			log.error("This is an error with placehold \"{}\"", "placeholder", e);
+			log.error("It also works with lambdas: {}", () -> "placeholder", () -> e);
```

Comment:
een object (zoals `e`) kan je direct meegeven :)
Je test hier verder niets. Is het handig dit er in te laten?

Reply:
Deze test hoeft niet bewaard te blijven, wat mij betreft.

Reply:
Ik heb hem weggehaald.